app/views/layouts/_admin_menu.html.haml

```diff
@@ -3,15 +3,15 @@
 %li= link_to 'Admin Dashboard', admin_root_path
 %li= link_to 'View Feedback', admin_feedback_index_path
 %li= link_to 'New Announcement', new_admin_announcement_path
-- if current_user.is_admin_or_organiser?
+- if current_user.admin_or_organiser?
   %li= link_to 'New workshop', new_admin_workshop_path
   %li= link_to 'New sponsor', new_admin_sponsor_path
 %li= link_to 'New event', new_admin_event_path
 %li= link_to 'New monthly', new_admin_meeting_path
 %li= link_to 'New chapter', new_admin_chapter_path
 %li= link_to 'New group', new_admin_group_path
 %li= link_to "Review jobs (#{jobs_pending_approval})", admin_jobs_path
-- if current_user.has_role?(:organiser, :any)
+- if current_user.organised_chapters.any?
```

Comment:
Can we revert this? `organised_chapter` will retrieve all chapters that someone is an organiser on and apply `any?` whilst checking for the role will just check for a match in the permissions table, which is faster.

Reply:
@despo sure, makes sense